tests/cases/fourslash/docCommentTemplateInterfaces.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path='fourslash.ts' />
+
+/////*interfaceFoo*/
+////interface Foo {
+////    /*propertybar*/
+////    bar: any;
+////
+////    /*methodbaz*/
+////    baz(message: any): void;
+////}
+
+verify.docCommentTemplateAt("interfaceFoo", /*expectedOffset*/ 8,
+`/**
+ * 
+ */`);
+
+verify.emptyDocCommentTemplateAt("propertybar");

```

Comment:
Why not give us the above template in this case too?

Reply:
I can, it was just outside of the scope of this particular bug fix. I can go and add these too if we want.

Reply:
I think we could just return that template (with the current indentation) at any arbitrary location where we don't have specific handling.

Reply:
Any arbitrary location? I feel like it should always be leading up to a declaration or member of some sort.